.github/workflows/netlify.yaml

```diff
@@ -0,0 +1,44 @@
+name: netlify production
+
+on:
+  push:
+    branches: [master, develop]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+
+      - name: Setup cache
+        uses: actions/setup-cache@v1
+        with:
+          path: ~/.npm
+          key: ${{runner.os}}-modules-${{hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{runner.os}}-modules-
+            ${{runner.os}}-
+
+      - name: Install dependency
+        run: npm ci
+
+      - name: Lint
+        run: npm run lint
```

Comment:
We don't have this script yet so it will fail. I think it's fine to remove it since every commit is linted before a commit is made

Reply:
Ok, removed.